modules/assim.sequential/inst/NEFI/US_Syv/syv.sh

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0"?>
```

Comment:
This filename looks like it should be a shell script for running the Sylvania forecast, but instead it's a pecan.xml. And not the template, a single specific XML with a specific day's drivers. I suspect this is a mistake and the correct file accidentally got overwritten?

Reply:
Oh good catch, I updated that